java/ql/src/semmle/code/java/frameworks/android/SharedPreferences.qll

```diff
@@ -0,0 +1,52 @@
+/* Definitions related to `android.content.SharedPreferences`. */
+import semmle.code.java.Type
+
+/* The interface `android.content.SharedPreferences` */
+library class TypeSharedPreferences extends Interface {
```

Comment:
`library class` is deprecated -- use a module instead

Reply:
I've converted <code>library class</code> to a module. Please let me know if there are some issues since it's the first module I created.